infra/modules/azure_core_infra/_modules/github_runner/container_app_jobs.tf

```diff
@@ -0,0 +1,98 @@
+resource "azurerm_container_app_job" "container_app_job" {
```

Comment:
why don't we use https://github.com/pagopa/dx/tree/main/infra/modules/github_selfhosted_runner_on_container_app_jobs ?

Reply:
On second thought, the job shouldn’t be configured here at all. Since this is a module in the DX repository that can be invoked across different repositories, it is not a core element to be created initially, it should be created under resources instead!

Reply:
right!